packages/docs/src/pages/theming.mdx

```diff
@@ -168,13 +167,13 @@ The breakpoints can then be used to apply [responsive styles](/sx-prop/#responsi
 
 ## Configuration Flags
 
-The theme object can also include configuration options for Theme UI. The following keys can be used to enable and disable certain features.
+The theme object can also include configuration options for Theme UI, which are nested in the `config` object.
+The following keys can be used to enable and disable certain features.
 
 | Flag                       | Default             | Description                                                                                                                                 |
 | -------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
 | `useCustomProperties`      | `true`              | Enables CSS custom properties to help mitigate a flash of unstyled content during rehydration                                               |
-| `useRootStyles`            | `false`             | Adds styles defined in `theme.styles.root` to the `<html>` element along with `color` and `background-color`                                |
-| `useBodyStyles`            | `true`              | (Will be deprecated in future) Adds styles defined in `theme.styles.root` to the `<body>` element along with `color` and `background-color` |
+| `useRootStyles`            | `true`              | Adds styles defined in `theme.styles.root` to the `<html>` element along with `color` and `background-color`                                |
 | `initialColorModeName`     | `'default'`         | The key used for the top-level color palette in `theme.colors`                                                                              |
 | `useColorSchemeMediaQuery` | `false`             | Initializes the color mode based on the `prefers-color-scheme` media query                                                                  |
```

Comment:
`useColorSchemeMediaQuery` default should be `true`, right ?

Reply:
Definitely. Good catch.